README.md

```diff
@@ -965,13 +965,12 @@ Copyright &copy; 2013-2024, Team Pa11y and contributors
 [info-build]: https://github.com/pa11y/pa11y/actions/workflows/tests.yml
 [info-license]: LICENSE
 [info-node]: package.json
-[info-npm]: https://www.npmjs.com/package/pa11y
```

Comment:
I think we still need this? It's the link for the badge at the top of the README

Reply:
Good catch